crates/nu_plugin_ps/src/ps.rs

```diff
@@ -68,6 +70,15 @@ pub async fn ps(tag: Tag, long: bool) -> Result<Vec<Value>, ShellError> {
                 UntaggedValue::filesize(memory.vms().get::<information::byte>()),
             );
             if long {
+                if let Ok(cpu_time) = process.cpu_time().await {
+                    let user_time = cpu_time.user().get::<time::nanosecond>().round() as i64;
+                    let system_time = cpu_time.system().get::<time::nanosecond>().round() as i64;
+
+                    dict.insert_untagged(
+                        "cpu_time",
+                        UntaggedValue::duration(BigInt::from(user_time + system_time)),
```

Comment:
You might want to do this as `BigInt::from(user_time) + BigInt::from(system_time)` if BigInt supports addition like that. That way you can do the addition with the full BigInt support rather than on just i64

Reply:
Actually, I don't think it matters much. Max i64 is like 292 years of nanoseconds :)